src/js/utils/apiClient.jsx

```diff
@@ -97,6 +97,9 @@ export const handleError = (error) => {
       });
       break;
     default:
+      if (error?.code === 'ERR_NETWORK') {
```

Comment:
Can you try if this works?
![image](https://github.com/openboxes/openboxes/assets/83239466/3360dceb-a556-4ab9-9f09-d1584cba9e1e)

Reply:
@alannadolny thanks, I was searching for something like this, but I didn't get any IDE suggestions, so I assumed there are not static strings for that and I did not see a reason to create one on my own for that thing.

Reply:
Yeah I also thought about avoiding creating our own "errors", so I looked into Axios implementation if they allow us to use their "errors" :smile: